### PR TITLE
[ui] Job status panel: tooltips on individual allocs

### PR DESCRIPTION
**Pull request by philrenaud**
Updates to the latest Helios (Hashicorp Design System) and includes new `<Hds::Tooltip>` elements to allocations in the status panel, showing their `shortId`.

This does not add any tooltip to "grouped" allocation blocks (the "+4" and "+3" blocks in the screenshot below).

image

### CODE REVIEW

File: ui/app/components/job-status/failed-or-lost.hbs [hunk 1]
```diff
@@ -8,13 +8,9 @@
   <div class="failed-or-lost-links">
     {{#if @supportsRescheduling}}
       <span>
-        <span
-          class="tooltip multiline text-center"
-          role="tooltip"
-          aria-label="Allocations that have been rescheduled, on another node if possible, due to failure or manual restart"
-        >
+        <Hds::TooltipButton @text="Allocations that have been rescheduled, on another node if possible, due to failure or manual restart">
```
Comment (philrenaud) [inline ui/app/components/job-status/failed-or-lost.hbs:11]: Side-effect to update the tooltips here from custom-rolled ones to the official Helios tooltips.

**Review by juliezzhou — COMMENTED** [review]
lgtm!

File: ui/app/components/job-status/individual-allocation.hbs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<Hds::TooltipButton @text="{{get @allocation "shortId"}}" aria-label="Allocation" @extraTippyOptions={{hash trigger=(if (eq @status "unplaced") "manual")}}>
```
Comment (lgfa29) [inline ui/app/components/job-status/individual-allocation.hbs:1]: Copyright bot is going to be angry about the missing header 😄

**Review by lgfa29 — CHANGES_REQUESTED** [review]
Changes LGTM, but I ran into an error when running `make dev-ui`:

```
Sass Syntax Error (SassCompiler) in /var/folders/sp/tb0_nysd40qf4w5jdhxgs8440000gp/T/broccoli-77574aZB57CvmdynR/out-700-funnel//app/styles/components/tooltip.scss:14:15

Error: Undefined variable.
   ╷
14 │   background: $black;
   │               ^^^^^^
   ╵
  /var/folders/sp/tb0_nysd40qf4w5jdhxgs8440000gp/T/broccoli-77574aZB57CvmdynR/out-700-funnel/app/styles/components/tooltip.scss 14:15                  @use
  /var/folders/sp/tb0_nysd40qf4w5jdhxgs8440000gp/T/broccoli-77574aZB57CvmdynR/out-700-funnel/app/styles/@hashicorp/design-system-components.scss 43:1  @import
  /var/folders/sp/tb0_nysd40qf4w5jdhxgs8440000gp/T/broccoli-77574aZB57CvmdynR/out-700-funnel/app/styles/app.scss 11:9                                  root stylesheet


Stack Trace and Error Report: /var/folders/sp/tb0_nysd40qf4w5jdhxgs8440000gp/T/error.dump.30842f53940c5a14089c1686e757354d.log
make: *** [ember-dist] Error 1
```

I think our [`tooltip.scss`](https://github.com/hashicorp/nomad/blob/main/ui/app/styles/components/tooltip.scss) is conflicting with Helios [`tooltip.scss`](https://github.com/hashicorp/design-system/blob/main/packages/components/app/styles/components/tooltip.scss). 

I renamed our file to `legacy-tooltip.scss` to fix it.

**Pull request merged**